components/Post.vue

```diff
@@ -68,7 +82,7 @@ figure {
   width: 100%; /* Set a small width */
   height: 60%;
   display: inline-block;
-  background-position: top;
+  background-size: cover;
```

Comment:
Was this change intended to be included in this PR?

Reply:
@TeddTech and I talked a bit more about this on a call. The reason is to better display/position the images used for search results and news items. 

The way items appear in master
![Screen Shot 2020-02-05 at 1 27 52 PM](https://user-images.githubusercontent.com/556024/73871156-56744a00-481b-11ea-81da-2728cd6fe010.png)

However, because this is a separate issue to what is mentioned in https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/10 I'm requesting that it be removed from this PR and filed as a new Issue.

Reply:
I have filed a new issue relating to this problem at Query Images #34 The PR is also up at #35